author.rb

```diff
@@ -13,4 +13,13 @@ def add_item(item)
     @items << item
     item.add_author(self)
   end
+
+  def to_json(*_arg)
+    {
+      id: @id,
+      first_name: @first_name,
+      last_name: @last_name,
+      items: @items.map(&:to_json)
+    }.to_json
+  end
```

Comment:
This looks great my friend, however, I already implemented these functions in my commit, can you please remove it?